README.md

```diff
@@ -4,8 +4,14 @@ This is a terminal emulator. Why yet another? Because.
 
 Installation
 ------------
-To install it you will need [rustup.rs](https://rustup.rs), then you can
-install it with Cargo.
+To install it you will need [rustup.rs](https://rustup.rs).
+
+```shell
+curl https://sh.rustup.rs -sSf | sh
```

Comment:
Doesn't apply to Windows, and doesn't consider the need to source `~/.cargo/env`